src/main/java/org/fundacionjala/coding/nestor/DnaStrand.java

```diff
@@ -0,0 +1,38 @@
+package org.fundacionjala.coding.nestor;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * Class to resolve complementary DNA.
+ */
+public class DnaStrand {
+  /**
+   * Method make complement.
+   *
+   * @param dna type String.
+   * @return return a string with dna complementary.
+   */
+  public String makeComplement(final String dna) {
+    return dna.chars()
+        .mapToObj(c -> String.valueOf((char) c))
+        .map(c -> complement(c))
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)